airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -1011,6 +1011,7 @@ def __init__(
         deferrable: bool = conf.getboolean("operators", "default_deferrable", fallback=False),
         **kwargs,
     ):
+        # FIXME: Handle if statement(s) in execute() method to comply with validate-operators-init pre-commit
         if waiter_check_interval_seconds is NOTSET:
```

Comment:
I'm sure that this not required fixed,`NOTSET` it is special sentinel which should not provided by user manually.